docs/en/README.md

```diff
@@ -28,32 +28,33 @@ Singularity supports a built-in encryption solution which encrypts the files wit
 
 ## Deal Making
 
-Singularity currently needs you to bring your own list of storage providers who are willing to accept your deals. There are two modes for deal making, a push mode where Singularity clients define how to send out deals - whether it's an one-off batch or a scheduled cron. And a pull mode where Storage providers can ask for deal proposals themselves as long as it satisfy the policies set by Singualrity clients
+Singularity currently needs you to bring your own list of storage providers who are willing to accept your deals. There are two modes for deal making:
+
+* a push mode where Singularity clients define how to send out deals - whether it's an one-off batch or a scheduled cronjob.
+* a pull mode where Storage providers can ask for deal proposals themselves as long as it satisfy the policies set by Singularity clients.
 
 ### Push Mode
 
-Singularity clients can setup a schedule with storage providers to send out a set number of deals for each interval time. They can also configured the maximum deals or deal size for each schedule as well as how many in-flight deals are allowed. Singularity handles deal renewal when deal is slashed, expires or it disappears on chain for any reason.
+Singularity clients can set up a schedule with storage providers to send out a set number of deals for each interval time. They can also configure the maximum deals or deal size for each schedule as well as how many in-flight deals are allowed. Singularity handles deal renewal when a deal is slashed, expires or disappears on chain for any reason.
 
 ### Pull Mode
 
-Singularity allows storage providers to query for list of PieceCIDs that they can propose deals to themselves as long as those storage providers are allowlisted in the deal making self service policy. This way, storage providers have ultimate flexibility to control the deal flow within their sealing pipelines.
+Singularity allows storage providers to query for a list of PieceCIDs that they can propose as deals to themselves as long as those storage providers are allowlisted in the deal making self service policy. This way, storage providers have ultimate flexibility to control the deal flow within their sealing pipelines.
 
 ### Wallet Management
 
-Singualrity allows clients to import Filecoin wallet using private keys and associate multiple wallets with dataset. It regularly checks datacap of each wallet to load balance between different wallets.
-
-Singualrity also supports remote signer so the client can keep their wallet private key to their own.
+Singularity allows clients to import a Filecoin wallet using private keys and associate multiple wallets with datasets. It regularly checks datacap of each wallet to load balance between different wallets.
 
-###
+Singularity also supports remote signer so the client can keep their wallet private key known only to them.
```

Comment:
```suggestion
Singularity also supports remote signing so the client can keep their wallet private key outside of the system.
```

Reply:
Sounds good, thanks for the rephrasing as even my version was still a little awkward.